[rabbitmq bindings] Add note (#3580)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-bindings/rabbitmq.md

## Spec metadata fields

> > When a new RabbitMQ message gets published, all values from the associated metadata will be added to the message's header values.

Reviewer: ```suggestion
> When a new RabbitMQ message gets published, all values from the associated metadata are added to the message's header values.
```
